### ISSUE TITLE
Statistics page: net value over time is based on current value of the chosen currency

### ISSUE THREAD

**Issue by hjorthjort** [rotki/rotki#3477]
## Problem Definition

When you open the statistics page and look at the Net value over time graph, it maintains its exact shape even if you change your base currency. So for example, if I choose USD as my base currency, and then switch to ETH or BTC and look at the last year, I would expect to see a very different shape of the graph.

I don't know how the net value history is stored. But I'll assume that it's stored in some currency, like USD. So my historical net value is saved as snapshots of my USD net value, as a time series. 

Then the issue could be solved by taking the historical value of the chosen bae currency relative to USD and divide by that for each point in the time series. So for example, if ETH was at 1000 USD at one point, then my net value at that point should be divided by 1000 when displaying that point in history. If the current value is 3500 USD, then the final point on the time series should be divided by 3500.

**Comment by LefterisJP** [comment]
Yes this is a known issue. And I thought we had a warning for that somewhere, but we don't? @kelsos don't we have an issue for this? It rings a bell.

Yes @hjorthjort the value is saved in USD. And conversion is done I assume in the frontend with the current EUR/USD ratio if you do EUR. Getting accurate historical data for each of these currencies to USD may be non-trivial. Doable, but not super trivial.

If we don't have any other issue open for this let's keep this one and try to find a way to make this right @kelsos. It's a valid request.

**Comment by kelsos** [comment]
I was pretty sure we had an issue about this, but I can't find any from a quick search. It is indeed a known issue and the frontend does the conversion based on the current price. Let's keep this open.

**Comment by LefterisJP** [comment]
Hey @kelsos as discussed via a call perhaps it would make sense to put the warning in the statistics page too as long as this is not addressed.
